### PR TITLE
[CostManagement] Release beta version

### PR DESCRIPTION
**Pull request by dvbb**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by dvbb** [comment]
linked issue: https://github.com/Azure/azure-rest-api-specs/issues/23704

File: sdk/cost-management/Azure.ResourceManager.CostManagement/src/Customization/Models/CostManagementExportData.Serialization.cs [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                    if (property.Value.ValueKind == JsonValueKind.Null)
+                    {
+                        continue;
+                    }
+                    eTag = new ETag(property.Value.GetString());
+                    continue;
+                }
+                if (property.NameEquals("id"u8))
+                {
+                    id = property.Value.GetString().StartsWith("/") ? new ResourceIdentifier(property.Value.GetString()) : new ResourceIdentifier($"/{property.Value.GetString()}");
```
Comment (dvbb) [inline sdk/cost-management/Azure.ResourceManager.CostManagement/src/Customization/Models/CostManagementExportData.Serialization.cs:50]: Current existing operations just have only one occurred `ID Validator issue`(https://github.com/Azure/azure-rest-api-specs/issues/23704), temporary use customization workaround

File: sdk/cost-management/Azure.ResourceManager.CostManagement/samples/autorest.tests.md
Comment (ArcturusZhang) [inline sdk/cost-management/Azure.ResourceManager.CostManagement/samples/autorest.tests.md:1]: why we have this file but no samples?

Comment (dvbb) [reply sdk/cost-management/Azure.ResourceManager.CostManagement/samples/autorest.tests.md:1]: The sample generator failed even though I removed all autorest.md change
```
 Exception: The method BenefitRecommendations_List that extends ArmResource might not exist, we should always use the client.GetXXXs(scope) to
  get the collection, this does not have to be invoked on a resource
```

File: sdk/cost-management/Azure.ResourceManager.CostManagement/src/Customization/Models/CostManagementExportData.Serialization.cs (hunk 1 above)
Comment (ArthurMa1978) [reply sdk/cost-management/Azure.ResourceManager.CostManagement/src/Customization/Models/CostManagementExportData.Serialization.cs:50]: Only this id is not start with "/" ?

Comment (dvbb) [reply sdk/cost-management/Azure.ResourceManager.CostManagement/src/Customization/Models/CostManagementExportData.Serialization.cs:50]: Oh, I check the specs and found some operations still have ID validator issues but the test does not cover them. I will double check them.

**Review by ArcturusZhang — APPROVED** [review]

**Review by ArthurMa1978 — APPROVED** [review]

File: sdk/cost-management/Azure.ResourceManager.CostManagement/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure CostManagement management client library for .NET
+
+**[Describe the service briefly first.]**
```
Comment (ArthurMa1978) [inline sdk/cost-management/Azure.ResourceManager.CostManagement/README.md:3]: Replace this with:
The Cost Management APIs provide the ability to explore cost and usage data via multidimensional analysis, where creating customized filters and expressions allow you to answer consumption-related questions for your Azure resources. These APIs are currently available for Azure Enterprise customers.

File: sdk/cost-management/ci.mgmt.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    - release/*
+  paths:
+    include:
+    - sdk/costmanagement/ci.mgmt.yml
+    - sdk/costmanagement/Azure.ResourceManager.CostManagement/
+
+extends:
+  template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: costmanagement
```
Comment (ArthurMa1978) [inline sdk/cost-management/ci.mgmt.yml:20]: The folder name is not same

**Pull request merged**